containers/src/Kokkos_DualView.hpp

```diff
@@ -164,14 +164,19 @@ class DualView : public ViewTraits<DataType, Arg1Type, Arg2Type, Arg3Type> {
   using t_host_const_randomread_um =
       typename t_dev_const_randomread_um::HostMirror;
 
-  //@}
-  //! \name Counters to keep track of changes ("modified" flags)
-  //@{
-
- protected:
-  // modified_flags[0] -> host
-  // modified_flags[1] -> device
-  using t_modified_flags = View<unsigned int[2], LayoutLeft, Kokkos::HostSpace>;
+  constexpr const static bool has_two_allocations =
```

Comment:
I've got no real preference, I'll let people discuss and go with whatever people like

Reply:
I hate to say "pick a different name" when I don't have an actual suggestion for one, but `has_two_allocations` is describing a mechanism where it would be better to describe the purpose it would be queried.

Reply:
I mean, we're querying whether it has two allocations because that's what matters, @jczhang07 has an Unmanaged DualView, and if it has two allocations he wants to free both, otherwise he only wants to free one.

Reply:
```c++
free(dv.dev_view)
if(dv.has_two_allocations) {
  free(dv.host_view);
}
```

Is where I got the name

Reply:
I suggest `has_same_space`.

Reply:
I'm completely on board with that, but it feels like we're going against what Nevin is saying above, we don't care that we have the same space, we care whether we have two allocations. Please excuse my pedantry here, I just want to make sure we have something everybody is comfortable with

Reply:
You made the case for `has_two_allocations`.